src_py/surfarray.py

```diff
@@ -63,7 +63,6 @@
 # Try to import the necessary modules.
 import pygame._numpysurfarray as numpysf
 
-from pygame.pixelcopy import array_to_surface, make_surface as pc_make_surface
```

Comment:
I'm not sure about this one.

They're not documented as being here, but I did find at least one person who was using surfarray.array_to_surface: https://github.com/search?q=%22pygame.surfarray.array_to_surface%22&type=Code

Reply:
I've re-added these imports. Thanks for reviewing the changes.

Reply:
You're welcome. Thanks for the update :)